src/sidebar/store/modules/test/selection-test.js

```diff
@@ -28,37 +28,51 @@ describe('sidebar/store/modules/selection', () => {
     });
   });
 
-  describe('setForceVisible()', function () {
+  describe('setForcedVisible()', function () {
```

Comment:
```suggestion
  describe('setForcedVisible', function () {
```

describe blocks shouldn't include parens, just the function name.